sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -61,7 +61,36 @@ private[ui] class AllExecutionsPage(parent: SQLTab) extends WebUIPage("") with L
           details.parentNode.querySelector('.stage-details').classList.toggle('collapsed')
         }}
       </script>
-    UIUtils.headerSparkPage("SQL", content, parent, Some(5000))
+    val summary: NodeSeq =
+      <div>
+        <ul class="unstyled">
+          {
+            if (listener.getRunningExecutions.nonEmpty) {
+              <li>
+                <a href="#running-execution-table"><strong>Running Queries:</strong></a>
+                {listener.getRunningExecutions.size}
+              </li>
+            }
+          }
+          {
+            if (listener.getCompletedExecutions.nonEmpty) {
+              <li id="completed-summary">
```

Comment:
I think `completed-summary` is for ui test on jobs and stages pages. We may no need to use it here if no test for it.

Reply:
I'm sorry, I do not quite understand what you mean. Here is just show statistics. Let the SQL UI and other UI interface style consistent.

Job ui:
![1](https://user-images.githubusercontent.com/26266482/30898443-f4f13c9a-a38d-11e7-8859-26f22ce53667.png)

Reply:
I can't find the style for `completed-summary`. Do we have style for it?

Reply:
I said the style is to show how much.

1.master ui:
![1](https://user-images.githubusercontent.com/26266482/30898687-25452766-a38f-11e7-97da-4fa1fef93dcf.png)

2.worker ui:
![2](https://user-images.githubusercontent.com/26266482/30898694-2c9bde74-a38f-11e7-831a-9b2ba81555d4.png)

3.job ui:
![4](https://user-images.githubusercontent.com/26266482/30898701-387e1c0c-a38f-11e7-905e-f5e391eb9d90.png)

4.stage ui:
![3](https://user-images.githubusercontent.com/26266482/30898709-3fbec084-a38f-11e7-9661-7677f6b7d82b.png)

Reply:
I meant `id=completed-summary` looks like is basically used for ui test against jobs and stages pages. I can't find we actually defined a style for it. If you don't add a test for it in this execution page, we may not need to add it.

Reply:
I am not talking this kind of style. I meant any css style defined for `id=completed-summary`...

Reply:
I understand what you mean. I will delete it. Thank you.

Reply:
Simply said, I think we can just write `<li>` instead of `<li id="completed-summary">` at this line.